dialogflow/spec/context_spec.rb

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "rspec"
+require "google/cloud/dialogflow"
+
+require_relative "../context_management"
+require_relative "../detect_intent_texts"
+
+describe "Context Management" do
+
+  before do
+    @project_id = ENV["GOOGLE_CLOUD_PROJECT"]
+    @session_id = "fake_session_for_testing"
+    @context_id = "fake_context_for_testing"
+  end
+
+  example "create context" do
+    # create a session
+    detect_intent_texts project_id:    @project_id,
+                        session_id:    @session_id,
+                        texts:         ["hi"],
+                        language_code: "en-US" 
+    expect {
+      create_context project_id: @project_id,
+                     session_id: @session_id,
+                     context_id: @context_id
+    }.to output(
```

Comment:
This tests that create_context outputs the right STDOUT but doesn't necessarily prove that the context exists (which checking List output is very helpful for)

Reply:
done